src/broadcast.jl

```diff
@@ -4,46 +4,93 @@
 
 import Base.Broadcast:
 BroadcastStyle, AbstractArrayStyle, Broadcasted, DefaultArrayStyle, materialize!
+import Base.Broadcast: combine_axes, instantiate, _broadcast_getindex, broadcast_shape, Style
```

Comment:
I'd suggest using `using` instead of `import` if no new methods are added to a function.

Reply:
Cleaned.